TypeError on triggering mqtt zone cleanup

<!-- Text between these tags doesn't get displayed so you don't have to remove it -->
## Describe the bug
<!--
A clear and concise description of what the bug is.
-->
Triggering a zoned cleanup causes an exception:
`
[2021-10-03T19:11:09.314Z] [ERROR] MQTT: Error while handling valetudo/staubsauger/ZoneCleaningCapability/start/set {
  payload: 'a7299e33-a926-485f-a0fe-fc0b7fd916dd',
  error: TypeError: loadedZone.includes is not a function
      at PropertyMqttHandle.setter (/snapshot/Valetudo/backend/lib/mqtt/capabilities/ZoneCleaningCapabilityMqttHandle.js:51:36)
      at PropertyMqttHandle.set (/snapshot/Valetudo/backend/lib/mqtt/handles/PropertyMqttHandle.js:108:27)
      at PropertyMqttHandle.setHomie (/snapshot/Valetudo/backend/lib/mqtt/handles/MqttHandle.js:329:20)
`

## To Reproduce
<!--
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->
Trigger a mqtt zone cleanup with the following content:
`
{
    "topic": "valetudo/staubsauger/ZoneCleaningCapability/start/set",
    "payload": "a7299e33-a926-485f-a0fe-fc0b7fd916dd"
}
`

### Vacuum Model
<!-- e.g. Roborock S5/Dreame D9/etc -->
Roborock S5

### Valetudo Version
2021.10.0

## Expected behavior
<!--
A clear and concise description of what you expected to happen.
-->
The zoned cleanup should work without an exception as it did with 2021.09.1.

## Additional context
This commit introduced the issue: https://github.com/Hypfer/Valetudo/commit/fdabcc52be3553f01648997ec714133e27a6e7cc#diff-1bea49ce162395621e0a70b8b5f1e31fc13651b423544e9e94ccfcd762e00ebd

LoadedZone is undefined in case of an invalid zone id or contains the zone so that the "includes" call that comes from the old code does not work here anymore.

![image](https://user-images.githubusercontent.com/41320207/135768665-cec4c728-4e62-450b-9af4-fc8278fff159.png)

--- a/backend/lib/mqtt/capabilities/ZoneCleaningCapabilityMqttHandle.js
+++ b/backend/lib/mqtt/capabilities/ZoneCleaningCapabilityMqttHandle.js
@@ -48,8 +48,8 @@ class ZoneCleaningCapabilityMqttHandle extends CapabilityMqttHandle {
                 format: "json",
                 setter: async (value) => {
                     const loadedZone = this.robot.config.get("zonePresets")[value];
-                    if (loadedZone.includes(undefined)) {
-                        throw new Error("Invalid zone ID found in start payload");
+                    if (!loadedZone) {
+                        throw new Error("Error while starting zone cleanup. There is no zone preset with id " + value);
                     }
                     try {
                         await this.capability.start(loadedZone.zones);

